src/interface.jl

```diff
@@ -244,7 +260,57 @@ isconstant(::Union{
           ) = true
 isconstant(L::AbstractSciMLOperator) = all(isconstant, getops(L))
 
-#islinear(L) = false
+"""
+    isconvertible(L) -> Bool
+
+Checks if `L` can be cheaply converted to an `AbstractMatrix` via eager fusion.
+"""
+isconvertible(L::AbstractSciMLOperator) = all(isconvertible, getops(L))
+
+isconvertible(::Union{
+                      # LinearAlgebra
+                      AbstractMatrix,
+                      UniformScaling,
+                      Factorization,
+
+                      # Base
+                      Number,
+
+                      # SciMLOperators
+                      AbstractSciMLScalarOperator,
+                     }
+             ) = true
+
+"""
+    concretize(L) -> AbstractMatrix
+
+    concretize(L) -> Number
+
+Convert `SciMLOperator` to a concrete type via eager fusion. This method is a
+no-op for types that are already concrete.
+"""
+concretize(L::Union{
+                    # LinearAlgebra
+                    AbstractMatrix,
+                    Factorization,
+
+                    # SciMLOperators
+                    AbstractSciMLOperator,
+                   }
+          ) =  convert(AbstractMatrix, L)
```

Comment:
```suggestion
                    AbstractArray,
                    Factorization,

                    # SciMLOperators
                    AbstractSciMLOperator,
                   }
          ) =  convert(AbstractArray, L)
```

Reply:
Should that be extended to array?

Reply:
And, in the package, I've defined `convert(::Type{AbstractMatrix}, L::MyOperator)`  methods for each operator type like

```julia
Base.convert(::Type{AbstractMatrix}, ii::IdentityOperator) = Diagonal(ones(Bool, ii.len))
```

So calling convert with AbstractArray would lead to errors:
```julia
julia> convert(AbstractArray, DiagonalOperator(ones(4)))                                                                                                                                                 [0/110]
ERROR: MethodError: Cannot `convert` an object of type 
  MatrixOperator{Float64, LinearAlgebra.Diagonal{Float64, Vector{Float64}}, SciMLOperators.FilterKwargs{typeof(SciMLOperators.DEFAULT_UPDATE_FUNC), Tuple{}}, SciMLOperators.FilterKwargs{typeof(SciMLOperators.DEFAULT_UPDATE_FUNC), Tuple{}}} to an object of type  
  AbstractArray
```